linux_sql/README.md

```diff
@@ -1,2 +1,38 @@
-#Linux Cluster Monitoring Agent
-This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Review.
+# <h1>Linux Cluster Monitoring Agent
+
+## <h2>Introduction
+Linux Cluster Monitoring Agent aims to collect server hardware specifications and monitor node resource usages, such as CPU and Memory, and store data in postgres database.
+
+## <h2>Architecture
+The diagram below shows the overview of the project:
+  
+![image](https://raw.githubusercontent.com/jarviscanada/jarvis_data_eng_derek/feature_linux_sql/linux_sql/Picture/1.png)
```

Comment:
please store image to `./assets/image_name.png` and use relative math to reference

e.e.
`![architecture_diagram](./assets/architecture_diagram.png)`